Create Agenda & Meeting Minutes Doc for 12/17/25 TAC Meeting (#128)
PQCA/TAC: meeting-minutes/2025/2025-12-17-TAC.md

---

## Voting Representative Attendance (_Alphabetical by 1st name_)
### Premier Member Representatives

* [x] Brian Jarvis, Amazon Web Services Inc. [TAC Chair]
* [ ] ~~JP Lomas, QRL~~
* [ ] ~~Michael Maximilien, IBM~~
* [ ] ~~Nicklas Körtge, IBM Research Zurich~~
* [ ] ~~Norman Ashley, Cisco~~
* [ ] ~~Yarkin Doroz, NVIDIA~~

 ### Project Representatives
* [ ] ~~Matthias Kannwischer, TSR~~
* [x] Hanno Becker, PQCP

## Non-Voting Representative Attendance

### Member Companies
* [ ] ~~Andreas Schade, IBM~~
* [ ] ~~Joe Livingston, IBM~~

> * [ ] ~~Prevek Sharma, University of Waterloo~~

Reviewer: ```suggestion
* [ ] ~~Pravek Sharma, University of Waterloo~~
```

### LF Staff 
* [x] Christina Harter 
* [x] Hart Montgomery 
* [ ] ~~Min Yu~~
* [ ] ~~Ry Jones~~

### Other Attendees
* [x] Aditya Koranga, NgKore \[TAC Vice Chair\]
* [x] Sandip Dholakia

---
# Meeting Agenda
- Open GitHub issues
  - Any ideas for further Discord work - https://github.com/PQCA/TAC/issues/114
  - Proof of concept for LLM-driven social media posts - https://github.com/PQCA/TAC/issues/110
  - Any update on educational content - https://github.com/PQCA/TAC/issues/115
- Blog post updates
  - CH: New Blog Post Submission from PQCP Mentee (hold on publishing until the New Year)
  - CH: Matthias - any update on the mlkem-native OpenType integration blog post?
- Follow-ups on other action items
  - OQS: Any update regarding project metrics / LFX insights
  - PQCP: Mentorship timeline update
  - CBOMkit: Update on TSC setup items mentioned in last meeting's notes
  
# Discussion & Updates

### **Introduction of 1st Time Attendees**
None.

---

### **RWPQC Talks**

> The meeting had light attendance due to the holiday season, with only a few participants including Brian, Christina, Hart, Sandip, Hanno and Aditya. They discussed the status of RWPQC talks, with Hart noting that Matthias might have been contacted but details were unclear. Hanno expressed interest in giving a talk if he attends RWPQC in Taiwan, though Sandip mentioned SAP would not approve travel for such events.

Reviewer 2: ```suggestion
The meeting had light attendance due to the holiday season, with only a few participants including Brian, Christina, Hart, Sandip, Hanno and Aditya. They discussed the status of RWPQC talks, with Hart noting that Matthias might have been contacted but details were unclear. Hanno expressed interest in giving a talk if he attends RWPQC in Taiwan.
```

---

### **Discord Usage and Spam Management**

> The TAC discussed Discord usage across projects. Hanno raised concerns about spam in the Open Quantum Safe general chat. Brian and Hart discussed mitigation strategies, including the use of designated channels as honeypots and automated moderation responses.

Reviewer 2: ```suggestion
The TAC discussed Discord usage across projects. Hanno raised concerns about spam in the Open Quantum Safe general chat. Brian and Hart discussed mitigation strategies, and a best practice which is to have the general channel - which will naturally attract more spam - but then spin off more narrowly-focused channels for more direct collaboration when needed which tend to have less spam.
```

The team discussed educational content development, with Hart reporting progress on a practical training program for PQC migrations targeting practicing developers, which will include a certification component. Hart mentioned a meeting with the LF Training and Education team scheduled for mid-January, with an expected update at the end of January. 

Sandip offered to help with the training content development based on his background in cryptography and technical writing, and agreed to review the Google document and email Hart offline with his suggestions.

---

### **Blog Updates and Mentorship Planning**

The team discussed several updates and follow-ups regarding blog posts and mentorship programs.

Christina announced that the review for a new blog post related to mldsa-native from a PQCP mentee remains open through next Monday, with planned publication on January 6, following the holidays.

The team also discussed the upcoming summer mentorship program timeline, with plans to open mentorship project proposals in early February and mentee applications in April. 

---

### **Project Updates**
- OQS
  - 
No OQS representatives.

- PQCP
  - 

>  Hanno provided updates on PQCP, noting progress on ML Chem native verification and functional correctness proofs for ARM and AVX2 assembly. Hanno also reported that PQCP had restructured their CI pipeline to reduce costs, with ongoing discussions with Rai about potential Amazon credits for open source projects.

Reviewer: ```suggestion
 Hanno provided updates on PQCP, noting progress on mlkem-native verification and functional correctness proofs for ARM and AVX2 assembly. Hanno also reported that PQCP had restructured their CI pipeline to reduce costs, with ongoing discussions with Ry about potential Amazon credits for open source projects.
```

---

## Voting Representative Attendance (_Alphabetical by 1st name_)
### Premier Member Representatives

* [ ] Brian Jarvis, Amazon Web Services Inc. [TAC Chair]
* [ ] JP Lomas, QRL
* [ ] Michael Maximilien, IBM
* [ ] Nicklas Körtge, IBM Research Zurich
* [ ] Norman Ashley, Cisco
* [ ] Yarkin Doroz, NVIDIA

 ### Project Representatives
* [ ] Matthias Kannwischer, TSR

## Non-Voting Representative Attendance

### Member Companies
* [ ] Andreas Schade, IBM
* [ ] Joe Livingston, IBM 
* [ ] Prevek Sharma, University of Waterloo

### LF Staff 
* [ ] Christina Harter 
* [ ] Hart Montgomery 
* [ ] Min Yu
* [ ] Ry Jones

### Other Attendees
* [ ] Aditya Koranga, NgKore \[TAC Vice Chair\]
* [ ] *Enter other attendees*

---
# Meeting Agenda
*To be entered by TAC Chair before each meeting*

- CH: New Blog Post Submission from PQCP Mentee (hold on publishing until the New Year)

> - CH: Matthias - any update on the ML native OpenType integration blog post?

Reviewer 2: ```suggestion
# Meeting Agenda
- Open GitHub issues
  - Any ideas for further Discord work - https://github.com/PQCA/TAC/issues/114
  - Proof of concept for LLM-driven social media posts - https://github.com/PQCA/TAC/issues/110
  - Any update on educational content - https://github.com/PQCA/TAC/issues/115
- Blog post updates
  - CH: New Blog Post Submission from PQCP Mentee (hold on publishing until the New Year)
  - CH: Matthias - any update on the ML native OpenType integration blog post?
- Follow-ups on other action items
  - OQS: Any update regarding project metrics / LFX insights
  - PQCP: Mentorship timeline update
  - CBOMkit: Update on TSC setup items mentioned in last meeting's notes
```

  
# Discussion & Updates

### **Introduction of 1st Time Attendees**
xxx

---

### **Discussion Item 1**
xxx

---

### **Project Updates**

> xxx

Reviewer 2: ```suggestion
- OQS
  - 
- PQCP
  - 
- CBOMkit
  - 

```

---

## Voting Representative Attendance (_Alphabetical by 1st name_)
### Premier Member Representatives

* [ ] Brian Jarvis, Amazon Web Services Inc. [TAC Chair]
* [ ] JP Lomas, QRL
* [ ] Michael Maximilien, IBM
* [ ] Nicklas Körtge, IBM Research Zurich
* [ ] Norman Ashley, Cisco
* [ ] Yarkin Doroz, NVIDIA

 ### Project Representatives
* [ ] Matthias Kannwischer, TSR

## Non-Voting Representative Attendance

### Member Companies
* [ ] Andreas Schade, IBM
* [ ] Joe Livingston, IBM 
* [ ] Prevek Sharma, University of Waterloo

### LF Staff 
* [ ] Christina Harter 
* [ ] Hart Montgomery 
* [ ] Min Yu
* [ ] Ry Jones

### Other Attendees
* [ ] Aditya Koranga, NgKore \[TAC Vice Chair\]
* [ ] *Enter other attendees*

---
# Meeting Agenda
- Open GitHub issues
  - Any ideas for further Discord work - https://github.com/PQCA/TAC/issues/114
  - Proof of concept for LLM-driven social media posts - https://github.com/PQCA/TAC/issues/110
  - Any update on educational content - https://github.com/PQCA/TAC/issues/115
- Blog post updates
  - CH: New Blog Post Submission from PQCP Mentee (hold on publishing until the New Year)

>   - CH: Matthias - any update on the ML native OpenType integration blog post?

Reviewer 2: ```suggestion
  - CH: Matthias - any update on the mlkem-native OpenType integration blog post?
```
